.bettercodehub.yml

```diff
@@ -0,0 +1,6 @@
+component_depth: 2
```

Comment:
Do we need to go further then 2, I see that `Client.Web/src/components/` has more then 2 depth?

Reply:
2 is enough, gets to the main components which is what this needs.